app/renderers/barcode_attribute_renderer.rb

```diff
@@ -0,0 +1,5 @@
+class BarcodeAttributeRenderer < Hyrax::Renderers::AttributeRenderer
+  def label
```

Comment:
We'll have to figure that out later. Hyrax has spanish labels now for properties, so I wonder if it's our overrides causing problems.